[2024-09-24T16:11:33.490+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-24T16:11:33.502+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_forest_classifier_dag.save_metrics_to_s3 manual__2024-09-24T16:08:20.328417+00:00 [queued]>
[2024-09-24T16:11:33.508+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_forest_classifier_dag.save_metrics_to_s3 manual__2024-09-24T16:08:20.328417+00:00 [queued]>
[2024-09-24T16:11:33.510+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-24T16:11:33.518+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_metrics_to_s3> on 2024-09-24 16:08:20.328417+00:00
[2024-09-24T16:11:33.528+0000] {standard_task_runner.py:64} INFO - Started process 7688 to run task
[2024-09-24T16:11:33.530+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'random_forest_classifier_dag', 'save_metrics_to_s3', 'manual__2024-09-24T16:08:20.328417+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/.ipynb_checkpoints/train_dag-checkpoint.py', '--cfg-path', '/tmp/tmpt9s7r2xv']
[2024-09-24T16:11:33.533+0000] {standard_task_runner.py:91} INFO - Job 328: Subtask save_metrics_to_s3
[2024-09-24T16:11:33.575+0000] {task_command.py:426} INFO - Running <TaskInstance: random_forest_classifier_dag.save_metrics_to_s3 manual__2024-09-24T16:08:20.328417+00:00 [running]> on host 82997fc6deb7
[2024-09-24T16:11:33.636+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='random_forest_classifier_dag' AIRFLOW_CTX_TASK_ID='save_metrics_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T16:08:20.328417+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-24T16:08:20.328417+00:00'
[2024-09-24T16:11:33.638+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-24T16:11:33.646+0000] {train_dag-checkpoint.py:171} INFO - Save metrics...
[2024-09-24T16:11:33.654+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-09-24T16:11:33.655+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-09-24T16:11:34.257+0000] {train_dag-checkpoint.py:201} INFO - DONE!!!
[2024-09-24T16:11:34.260+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-24T16:11:34.260+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-24T16:11:34.270+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_forest_classifier_dag, task_id=save_metrics_to_s3, run_id=manual__2024-09-24T16:08:20.328417+00:00, execution_date=20240924T160820, start_date=20240924T161133, end_date=20240924T161134
[2024-09-24T16:11:34.307+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-24T16:11:34.322+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-24T16:11:34.326+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
